README.md

```diff
@@ -0,0 +1,56 @@
+# eslint-config-feedzai
+
+This package provides the ESLint configurations as an extensible shared configuration that could be used by everyone and across the Feedzai's products.
+
+This repo includes two distinct ESLint configurations:
+
+- `eslint-config-feedzai`- This package contains our ESLint rules, including the ones for ECMAScript 6+. 
```

Comment:
This configuration contains most of our rules, except the React related ones, including...

Calling configurations packages might be confusing.